service/rest_version.go

```diff
@@ -580,16 +580,11 @@ func (ra *restAPI) lastGreen(w http.ResponseWriter, r *http.Request) {
 	// GET /rest/v1/projects/mongodb-mongo-master/last_green?linux-64=1&windows-64=1
 	queryParams := r.URL.Query()
 
-	// Make sure all query params are valid variants and put them in an array
+	// Don't validate build variants are in project, since they may be generated
+	// variants that don't yet exist for the latest parser project.
```

Comment:
Are we opening it up for potential errors down the line if they don't end up being generated?

Reply:
The consequence would be that the query returns nothing, if the build variant actually doesn't match any existing version. I do think this is reasonable behavior.